rust/ql/lib/codeql/rust/Concepts.qll

```diff
@@ -104,6 +119,13 @@ module RemoteSource {
   }
 }
 
+/**
+ * An externally modeled source for remote (network) data.
+ */
+class ModeledRemoteSource extends RemoteSource::Range {
+  ModeledRemoteSource() { sourceNode(this, "remote-source") }
```

Comment:
I believe other languages simply use `"remote"`, e.g. https://github.com/github/codeql/blob/75a80f2a3635072253a69931e6af802d3f75b25c/cpp/ql/lib/ext/Boost.Asio.model.yml#L7. That being said, seeing as we also have `"remote-sink"`, perhaps `"remote-source"` is better...

Reply:
I think consistency with other languages is important here, so I've changed it to just `"remote"`.  (I think `"remote-sink"` is only used by CPP FWIW)

Reply:
> I think consistency with other languages is important here

Agreed.